packages/core/package.json

```diff
@@ -73,7 +73,7 @@
     "deepmerge": "^4.2.2",
     "hoist-non-react-statics": "^3.3.2",
     "moize": "^5.4.7",
-    "pcln-icons": "4.0.0",
+    "pcln-icons": "4.0.1-holiday-2020.1",
```

Comment:
This was necessary to apply the latest changes from `pcln-icons` due to the cyclic dependency between `pcln-design-system` and `pcln-icons`. It should be inconsequential as it will be replaced when we apply changes prior to the next release.